### PR TITLE
Shrinking spotlight model

### PR DESCRIPTION
**Pull request by AndrewZhang599**
Added the shrinking spotlight model and drift function config. Also a new cython simulator without v.

### CODE REVIEW

File: src/cssm.pyx [hunk 1]
```diff
@@ -870,7 +870,155 @@ def ddm_flex(np.ndarray[float, ndim = 1] v,
     else:
         raise ValueError('return_option must be either "full" or "minimal"')
 # ----------------------------------------------------------------------------------------------------
+def ddm_flex_no_v(
```
Comment (AlexanderFengler) [inline src/cssm.pyx:873]: question here: was there no way to reuse `ddm_flex` by e.g. setting `v` to `0`?

File: ssms/basic_simulators/simulator.py [hunk 2]
```diff
@@ -513,6 +513,7 @@ def simulator(
         "ornstein_uhlenbeck",
         "ornstein_angle",
         "gamma_drift",
+        "shrink_spot",
```
Comment (AlexanderFengler) [inline ssms/basic_simulators/simulator.py:516]: missing the part where you assign a particular `sigma` (`s` parameter).
can you confirm that sigma is set to the correct value for this simulator, and that the parameter bounds are the correct ones for that particular sigma?

**Review by AlexanderFengler — COMMENTED** [review]
Looks good just a few comments for verification.

File: ssms/basic_simulators/simulator.py (hunk 2 above)
Comment (AndrewZhang599) [reply ssms/basic_simulators/simulator.py:516]: Yes, the sigma value for shrink_spot is 1.0. The bounds that I have specified in config are the ones specified in the paper, where p_target/outer/inner are scaled by 10. "a" was already between 0.3 and 3.0, which covered the bounds specified in that paper already.

File: src/cssm.pyx (hunk 1 above)
Comment (AndrewZhang599) [reply src/cssm.pyx:873]: I think the issue was that there's no way to set "v" equal to 0 in hssm. I also would need to specify bounds in config for "v" even though I don't want it to be simulated. I could try to add an if statement in simulator.py that just sets theta["v"] = 0, though id still have to specify the bounds and "estimate" it in hssm.

**Review by AlexanderFengler — APPROVED** [review]
Ok to merge.

**Pull request merged**